webui/react/src/components/kit/LogViewer/LogViewer.test.tsx

```diff
@@ -279,7 +279,7 @@ describe('LogViewer', () => {
           const lastLog = logsReference[logsReference.length - 1];
           expect(screen.queryByText(lastLog.message)).toBeInTheDocument();
         },
-        { timeout: 4000 },
+        { timeout: 6000 },
```

Comment:
The test has an overall timeout of 5000ms, so you'll need to bump the timeout there as well.